Jetpack Backup: Prevent restores with invalid creds (#68552)
Automattic/wp-calypso: client/components/data/query-jetpack-credentials-status/README.md

# Query Jetpack Credentials Status

> `<QueryJetpackCredentialsStatus />` is a React component which dispatches actions for testing Jetpack Credentials to determinate if they are still valid or not.

Reviewer: Left a bit of suggestions for improvement here:

```suggestion
`<QueryJetpackCredentialsStatus />` is a React component that dispatches actions for testing Jetpack Credentials to determine whether they are still valid.
```

Author: Sounds reasonable, thanks for that one!
